daemon/wiretap_service_test.go

```diff
@@ -0,0 +1,122 @@
+package daemon
+
+// this test does not work. I have no idea how to test the platform. It will take a decent amount of work to dig deep.
```

Comment:
You don't need to test the platform, you need to test the logic, all can be  mocked using the standard library.

mockedhttps://www.cloudbees.com/blog/testing-http-handlers-go